### PR TITLE
preserve ad markers when transition from V2L->Live

### PR DESCRIPTION
**Pull request by oscnord**
Fix: #99 

This PR adds support for cue tags when transitioning from V2L->Live. 

If an `EXT-X-CUE-OUT` (with a duration) is included in the VOD manifest it will be preserved in the transition manifest.

### CODE REVIEW

File: engine/session_live.js
Comment (oscnord) [reply engine/session_live.js]: Yeah that is not needed, I have updated this in the latest commit thanks for pointing that out. 
The cue tags are separate in an attribute object per segment.

```
{
  "attributes": {
    "attributes": {
      "cuein": true
    }
  },
  "properties": {
    "byteRange": null,
    "daiPlacementOpportunity": null,
    "date": null,
    "discontinuity": null,
    "duration": 4.24,
    "title": "",
    "uri": "https://example.com/video.ts"
  }
}
```

**Comment by oscnord** [comment]
This needs a bit more testing before being merged

**Review by Nfrederiksen — APPROVED** [review]
nice

**Comment by oscnord** [comment]
An update is needed for https://github.com/Eyevinn/hls-vodtolive to make it possible for the transition manifest to contain cue tags when going from Live -> V2L.

**Comment by birme** [comment]
> An update is needed for https://github.com/Eyevinn/hls-vodtolive to make it possible for the transition manifest to contain cue tags when going from Live -> V2L.

hls-vodtolive v1.16.4 now available including the above patch

**Review by Nfrederiksen — APPROVED** [review]
Great. This will make sure we have consistent segment amount in the manifests.

**Comment by oscnord** [comment]
@birme you can merge this if everything looks good. :)

**Review by birme — APPROVED** [review]

**Pull request merged**
